### DISCUSSION TITLE
v4.8

### DISCUSSION THREAD

**Discussion by port19x** [pystardust/ani-cli#1278] [General] [answered]
My sincere apologies to everyone on the last release which is presumably broken for a while at this point.
Cause for breakage was removal of curls `--cipher` arg which we wrongly assumed to only affect windows users.
This only came to my attention in #1275, so big thanks to @Technetium1 for pointing it out.
I'll review our CI & maintenance protocol to prevent this in the future

## What's Changed

- Documentation
  - homies section saiku -> dantotsu by @WanderRainID in https://github.com/pystardust/ani-cli/pull/1247
  - simplify (windows) docs by @71zenith in https://github.com/pystardust/ani-cli/pull/1273

- Bug Fixes
  - curl in gitbash doesn't support `--cipher` by @Derisis13 in https://github.com/pystardust/ani-cli/pull/1244
  - removed last --cipher argument by @Derisis13 in https://github.com/pystardust/ani-cli/pull/1252
  - Steam deck installation and .desktop fix by @Nannk in https://github.com/pystardust/ani-cli/pull/1269
  - correctly closes escape sequence in message 'Checking dependencies…' by @ochaton in https://github.com/pystardust/ani-cli/pull/1272
  - script speed improvement on windows by @CoolnsX in https://github.com/pystardust/ani-cli/pull/1262

- Features
  - add --skip-title <title> by @LeFlorians in https://github.com/pystardust/ani-cli/pull/1253
  - no-detach by @port19x in https://github.com/pystardust/ani-cli/pull/1264 
    *(broken until #1274, but can't wait any longer with this release)*

## New Contributors
* @WanderRainID made their first contribution in https://github.com/pystardust/ani-cli/pull/1247
* @LeFlorians made their first contribution in https://github.com/pystardust/ani-cli/pull/1253
* @ochaton made their first contribution in https://github.com/pystardust/ani-cli/pull/1272

**Full Changelog**: https://github.com/pystardust/ani-cli/compare/v4.7...v4.8

<hr /><em>This discussion was created from the release <a href='https://github.com/pystardust/ani-cli/releases/tag/v4.8'>v4.8</a>.</em>

**Comment by justchokingaround** [comment, marked as answer]
![image](https://github.com/pystardust/ani-cli/assets/44473782/3a1c13e6-d41d-4a26-a83e-d10ba4a6808b)
